prisma/schema.prisma

```diff
@@ -44,26 +46,25 @@ model Activity {
 model Sprint {
   id                Int       @id @default(autoincrement())
   title             String
-  start_date        DateTime
-  end_date          DateTime
-  planned_capacity  Int
+  start_date        DateTime?
+  end_date          DateTime?
+  planned_capacity  Int?
   project           Project  @relation(fields: [project_id], references: [id])
   project_id        Int
-  status            Status    @relation(fields: [status_id], references: [id])
-  status_id         Int
   tasks             Task[]  
 }
 
 model Task {
   id                  Int       @id @default(autoincrement())
   title               String
-  description         String
-  requirements        String
-  acceptance_criteria String
-  points              Int
-  user                User      @relation(fields: [assigned_to], references: [id])    
-  assigned_to         Int
-  sprints             Sprint   @relation(fields: [sprint_id], references: [id])
+  description         String?
+  category            String?    
+  requirements        String?
+  acceptance_criteria String?
+  points              Int?
+  user                User?     @relation(fields: [assigned_to], references: [id])    
+  assigned_to         Int?
+  sprint              Sprint    @relation(fields: [sprint_id], references: [id])
```

Comment:
Thanks for updating these to optional fields! I was just about to ask for that yesterday

Reply:
No problem!